src/main/common/interestUtils.ts

```diff
@@ -51,3 +56,19 @@ export async function draftClaimAmountWithInterest (claimDraft: DraftClaim): Pro
 
   return claimAmount + interest
 }
+
+function getInterestRate (claimDraft: DraftClaim): number {
+  if (claimDraft.interest.option === InterestOption.NO) {
+    return 0.00
+  }
+
+  if (claimDraft.interestRate.type === InterestRateOption.STANDARD) {
+    return getStandardInterestRate()
+  } else {
+    return claimDraft.interestRate.rate
+  }
+}
+
+export function getStandardInterestRate (): number {
```

Comment:
why do you need a function for this?
you can do an `export const` instead?
REQ